octokit/organizations.go

```diff
@@ -2,13 +2,101 @@ package octokit
 
 // Organization is a representation of an organization on GitHub, containing
 // all identifying information related to the specific organization.
+
+import (
+	"time"
+)
+
+var (
+	OrganizationReposURL = Hyperlink("/orgs/{org}/repos{?type,page,per_page,sort}")
+	OrganizationURL      = Hyperlink("/orgs/{org}")
+	YourOrganizationsURL = Hyperlink("/user/orgs")
+	UserOrganizationsURL = Hyperlink("/users/{username}/orgs")
+)
+
+// A service to return organization information
+type OrganizationService struct {
+	client *Client
+}
+
+// Get the user search results based on OrganizationService#URL
+func (g *OrganizationService) OrganizationRepos(uri *Hyperlink, params M) (
+	repos []Repository, result *Result) {
+	if uri == nil {
+		uri = &OrganizationReposURL
+	}
+	url, err := uri.Expand(params)
+	if err != nil {
+		return make([]Repository, 0), &Result{Err: err}
+	}
+	result = g.client.get(url, &repos)
+	return
+}
+
+// Get the user search results based on OrganizationService#URL
+func (g *OrganizationService) Organization(uri *Hyperlink, params M) (
+	organization Organization, result *Result) {
+	if uri == nil {
+		uri = &OrganizationURL
+	}
+	url, err := uri.Expand(params)
+	if err != nil {
+		return Organization{}, &Result{Err: err}
+	}
+	result = g.client.get(url, &organization)
+	return
+}
+
+// Get the issue search results based on OrganizationService#URL
+func (g *OrganizationService) YourOrganizations(uri *Hyperlink, params M) (
+	organizations []Organization, result *Result) {
+	if uri == nil {
+		uri = &YourOrganizationsURL
+	}
+	url, err := uri.Expand(params)
+	if err != nil {
+		return make([]Organization, 0), &Result{Err: err}
+	}
+	result = g.client.get(url, &organizations)
+	return
+}
+
+// Get the issue search results based on OrganizationService#URL
+func (g *OrganizationService) UserOrganizations(uri *Hyperlink, params M) (
+	organizations []Organization, result *Result) {
+	if uri == nil {
+		uri = &UserOrganizationsURL
+	}
+	url, err := uri.Expand(params)
+	if err != nil {
+		return make([]Organization, 0), &Result{Err: err}
+	}
+	result = g.client.get(url, &organizations)
+	return
+}
+
 type Organization struct {
```

Comment:
I think we'll need a [`Plan` struct][plan] for authenticated requests for users that can see that info.

[plan]: https://developer.github.com/v3/orgs/#response-3